### PR TITLE
Issue #2: Added javadoc comments for ImageFileFilter

### PR DESCRIPTION
**Pull request by FridaTveit**

### CODE REVIEW

File: src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java [hunk 1]
```diff
@@ -19,8 +19,16 @@
 import javax.swing.filechooser.FileFilter;
 import java.io.File;
 
+/**
+ * This class filters filenames using the filename extension
```
Comment (oskopek) [inline src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java]: Checkstyle: First sentence should end with a period.

File: src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java [hunk 2]
```diff
@@ -31,15 +39,25 @@ public static boolean accept(String name) {
...
+        if (file.isDirectory()) {
             return true;
         }
-        String name = f.getName();
+        String name = file.getName();
         return accept(name);
     }
 
+    /**
+     * This method returns a description of the file extensions which are currently accepted
```
Comment (oskopek) [inline src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java]: Checkstyle: First sentence should end with a period.

File: src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java (hunk 1 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java]: Okay, will change it!

File: src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java (hunk 2 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java]: Okay, will change it!

File: src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java (hunk 1 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java]: Done :)

File: src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java (hunk 2 above)
Comment (FridaTveit) [inline src/main/java/net/sf/javaanpr/gui/tools/ImageFileFilter.java]: Done :)

**Pull request merged**